### DISCUSSION TITLE
Here is a keymap finger placement color assist generator.

### DISCUSSION THREAD

**Discussion by eliezersouzareis** [monkeytypegame/monkeytype#5966] [Ideas]
When I learned touch typing on QWERTY, the site I used had a finger placement assist. After "mastering" touch typing on QWERTY, I became interested in switching to Dvorak. As I practiced on Monkeytype, I felt the need for a color assist for finger placement. Since I wasn't keen on doing the CSS manually, I wrote a generator.

[demo with qwerty.webm](https://github.com/user-attachments/assets/62311d94-d0c0-4e03-93a8-4d9fb217979e)

https://gist.github.com/eliezersouzareis/bfdc65b420649389cb0e3edc3a1bfa15
Use the generated CSS with an extension that overrides the site's CSS (such as Stylus).

Learning Dvorak didn't pan out for me, but I hope this can be helpful to anyone who needs something similar.

**Comment by eliezersouzareis** [comment]
I do recon this is somewhat a duplicate of this [discussion](https://github.com/monkeytypegame/monkeytype/discussions/5931), but my approach is different (that is, just highlight the key being typed / to be typed)

**Comment by alexchexes** [comment]
Is something wrong with the video or is it my browser?

![image](https://github.com/user-attachments/assets/32e607d4-91e5-40cc-a2be-3bd6275a5f53)

**Reply by eliezersouzareis** [reply]
I tried again using other format. I hope this works for you.

https://github.com/user-attachments/assets/8fa59203-66c0-4437-8a2d-2d006c35ff6f
